File: userGenerator/userGen.js
var helper = require("./helper.js");
var util = require("util");
var fs = require('fs');

var sql = "";
for (var i = 1001; i <= 10000; i++) {
	var password = helper.encryptPassword("12345", 20);
	sql += util.format("INSERT INTO `securityInfo` (`id`, `registerEmail`, `salt`, `password`) VALUES ('%s', 'test%s@test.com', '%s', '%s');\n",
		i, i, password.salt, password.password);
}
fs.appendFile("securityInfo.sql", sql, function (err) {
	if (err) console.log(err);
});

File: userGenerator/userGen.py
f = open("insertUser.sql", "w")
for x in range(1001, 10001):
	f.write("INSERT INTO `accounts` (`id`,`fid`,`registerEmail`,`username`) VALUES (\'"+str(x)+"\', \'0\', \'test"+str(x)+"@test.com\',\'tester"+str(x)+"\');\n")
	f.write("INSERT INTO `info` (`id`, `name`) VALUES (\'" + str(x) + "\', \'Test" + str(x) + " Testman\');\n")
